### PR TITLE
Update the Code of Conduct to the latest version

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Update the Code of Conduct to the latest version.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -114,12 +114,19 @@ the community.
...
-This Code of Conduct is adapted from the Contributor Covenant homepage, 
-version 2.0, available at
-[v2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct.html).
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.1, available at
+[https://www.contributor-covenant.org/version/2/1/code_of_conduct.html][v2.1].
 
-Community Impact Guidelines were inspired by Mozilla's code of conduct enforcement ladder.
+Community Impact Guidelines were inspired by
+[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
```
Comment (shuklaritvik06) [inline CODE_OF_CONDUCT.md:122]: But links should be []() like this ?

Comment (shuklaritvik06) [reply CODE_OF_CONDUCT.md:122]: That should be <[]()> like this right?

Comment (Panquesito7) [reply CODE_OF_CONDUCT.md:122]: Links can also be like that in Markdown. It can help in scenarios where the same link is used multiple times. 🙂

Comment (shuklaritvik06) [reply CODE_OF_CONDUCT.md:122]: Oh :)

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Pull request merged**